docusaurus/docs/Android/01-basics/02-dependencies.mdx

```diff
@@ -2,11 +2,25 @@
 
 All Stream Android libraries are available from MavenCentral, with some of their transitive dependencies hosted on Jitpack. 
 
-To add Stream dependencies, update your repositories in the project level `build.gradle` file to include these two repositories:
+Before you add Stream dependencies, update your repositories in the `settings.gradle` file to include these two repositories:
 
-```groovy
+```groovy {5-6}
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+        maven { url "https://jitpack.io" }
```

Comment:
is jitpack required?

Reply:
Unfortunately yes, we still depend on it: https://github.com/GetStream/stream-chat-android/blob/main/build.gradle#L37-L52

I'll revisit these dependencies next week though, maybe some can be retrieved from other places now.